services/user/XAdmin/ui/src/pages/create-page.tsx

```diff
@@ -156,47 +167,64 @@ export const CreatePage = () => {
     const [currentState, setCurrentState] = useState(1);
 
     useEffect(() => {
+        const setKeysAndBoot = async () => {
+            try {
+                let key: CryptoKey | undefined;
+                if (!isDev) {
+                    key = await createAndSetKey();
+                }
+                const desiredPackageIds = Object.keys(rows);
+                const desiredPackages = packages.filter((pack) =>
+                    desiredPackageIds.some((id) => id == getId(pack))
+                );
+                const requiredPackages = getRequiredPackages(
+                    packages,
+                    desiredPackages.map((pack) => pack.name)
+                );
+                bootChain(requiredPackages, bpName, (state) => {
```

Comment:
The server's public key needs to be passed to boot.

Reply:
Ah, thank you